pages/linux/semanage-boolean.md

```diff
@@ -0,0 +1,21 @@
+# semanage boolean
+
+> Manage persistent SELinux boolean settings.
+> See also: `semanage`, `getsebool`, `setsebool`.
+> More information: <https://manned.org/man/semanage-boolean>.
+
+- List all booleans settings:
+
+`sudo semanage boolean {{-l|--list}}`
+
+- List all user-defined boolean settings without headings:
+
+`sudo semanage boolean {{-l|--list}} {{-C|--locallist}} {{-n|--noheading}}`
+
+- Set or unset a boolean temporarily (non-persistent across reboot):
+
+`sudo setsebool {{httpd_can_network_connect}} {{1|true|on|0|false|off}}`
```

Comment:
Instead of example for another command, wouldn't be better to do something like ``See also `setsebool` for managing booleans temporarily``

Reply:
Yeah I actually agree with you on this. But how do we word this though (considering we also want to reference `semanage` and `getsebool`)?

Reply:
> Yeah I actually agree with you on this. But how do we word this though (considering we also want to reference `semanage` and `getsebool`)?

I added a suggestion.